LEAF_Nexus/sources/System.php

```diff
@@ -199,4 +199,17 @@ public function removeReportTemplate($in)
             }
         }
     }
+
+    public function refreshOrgchartEmployees()
+    {
+        $memberships = $this->login->getMembership();
+        if (!isset($memberships['groupID'][1]))
+        {
+            return 'Admin access required';
+        }
+        
+        header('Cache-Control: no-cache');
+        exec('php ../scripts/refreshOrgchartEmployees.php &', $output);
```

Comment:
@dastan-rahimi 

Will the use of the relative path here work in production? Note that existing pattern from mailer uses variable for path directory that is based on php variable __FILE__:

https://github.com/VHAINNOVATIONS/LEAF/blob/master/LEAF_Request_Portal/Email.php#L177

cc: @thomasnguyen95

Reply:
Yes it will work